src/main/java/practice/StreamPractice.java

```diff
@@ -13,8 +16,15 @@ public class StreamPractice {
      * If there is no needed data throw RuntimeException with message
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
+
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(n -> n.split(","))
+                .flatMap(Arrays::stream)
+                .map(Integer::parseInt)
```

Comment:
All magic numbers, strings, and symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
Don't use short names
```suggestion
        return numbers.stream()
                .map(line -> line.split(SEPARATOR))
                .mapToInt(Integer::parseInt)
```